solr/prometheus-exporter/src/java/org/apache/solr/prometheus/exporter/SolrExporter.java

```diff
@@ -266,6 +296,12 @@ public static void main(String[] args) {
         clusterId = defaultClusterId;
       }
 
+      if (!res.getString(ARG_USER_DEST).isEmpty() && !res.getString(ARG_PASSWORD_DEST).isEmpty()) {
+        String user = res.getString(ARG_USER_DEST);
+        String password = res.getString(ARG_PASSWORD_DEST);
+        scrapeConfiguration.withBasicAuthCredentials(user, password);
```

Comment:
i ended up littering the https://github.com/apache/solr/pull/1954 code iwth lots of things that look like this too ;-).